### DISCUSSION TITLE
Mosquitto on Mac Monterey

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#2836] [Getting started]
Hi all,

Mosquitto (mosquitto.org) gives the install instructions as a simple 'brew install mosquitto'. Running that gave me the following errors:
fatal: Could not resolve HEAD to a revision
Warning: No available formula with the name "mosquitto".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

Am I right in assuming Mosquitto is no longer available on brew?

I hate to be negative about brew - but the moment I see instructions to install something with brew I just know it won't work! It seems to me to have taken a complex task (installing software) and made it even more complex, then added some terrible metaphors to try and make it into beer related!

**Comment by neutric** [comment]
```
%  brew info mosquitto                                                                                                       01/22/2022 00:15:27 CET
mosquitto: stable 2.0.14 (bottled)
Message broker implementing the MQTT protocol
[...]
==> Analytics
install: 2,667 (30 days), 9,714 (90 days), 45,394 (365 days)
```
The problem seems a broken Homebrew configuration on your end. Please read https://docs.brew.sh/Troubleshooting.
